notebooks/algorithms/centrality/Eigenvector.ipynb

```diff
@@ -169,25 +178,7 @@
    ]
   },
   {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "### Read in the data - GPU\n",
-    "cuGraph depends on cuDF for data loading and the initial Dataframe creation\n",
-    "\n",
-    "The data file contains an edge list, which represents the connection of a vertex to another.  The `source` to `destination` pairs is in what is known as Coordinate Format (COO).  In this test case, the data is just two columns.  However a third, `weight`, column is also possible"
```

Comment:
not a big deal at all, but explaining COO form is useful even though using datasets makes it more transparent

Reply:
I'll go back and include a sentence or two about COO form even though the notebook was converted to use Datasets